src/components/navigation-tabs/BackToInstructor.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
+import { useParams } from 'react-router-dom';
 
 import { getConfig } from '@edx/frontend-platform';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { Button, Icon } from '@edx/paragon';
 import { ArrowBack } from '@edx/paragon/icons';
 
 export default function BackToInstructor() {
+  const { courseId } = useParams();
+
   return (
     <Button
       variant="tertiary"
       className="mb-4.5 ml-n4.5 text-primary-500"
-      href={`${getConfig().LMS_BASE_URL}/courses/${window.location.pathname.split('/')[2]}/instructor#view-course-info`}
+      href={`${getConfig().LMS_BASE_URL}/courses/${courseId}/instructor#view-course-info`}
```

Comment:
`courseId` should be passed in as a prop to this component.

It shouldn't be hard to do, as the only place that uses component has the variable [in scope](https://github.com/openedx/frontend-app-communications/blob/master/src/components/bulk-email-tool/BulkEmailTool.jsx#L16).